openapi/data_repository_service.swagger.yaml

```diff
@@ -164,6 +164,8 @@ definitions:
         type: string
         description: |-
           A string that can be used to name an `Object`.
+          This string is made up of uppercase and lowercase letters, decimal digits, hypen, period, and underscore [A-Za-z0-9.-_]. See https://tools.ietf.org/html/rfc3986#section-2.3[RFC 3986 § 2.3].
```

Comment:
Is there a reason that we want names to use the same characters as URIs? Maybe reference [POSIX filenames](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_282) instead?

Reply:
Let's talk about this tomorrow so I can be sure I understand what you mean

Reply:
I found this really helpful reference here: https://en.wikipedia.org/wiki/Filename#Comparison_of_filename_limitations